bindings/matrix-sdk-ffi/src/sliding_sync.rs

```diff
@@ -598,7 +598,7 @@ impl SlidingSyncList {
         &self,
         observer: Box<dyn SlidingSyncListRoomListObserver>,
     ) -> Arc<TaskHandle> {
-        let mut room_list_stream = self.inner.room_list_stream();
+        let (_, mut room_list_stream) = self.inner.room_list_stream();
```

Comment:
Why are we ignoring the current values here? I think we should return it, or immediately call `observer.did_receive_update(Append { values })` (or `Reset` instead of `Append`; at least if non-empty).

Reply:
It's already managed with the `SlidingSyncListBuilder::once_built` + Callback by ElementX. This FFI binding for SS is about to be removed, let's not modify something that works for now and that will finish in the trash in less than 4 weeks ;-).